templates/profile/hathitrust/solr6/log4j.properties.erb

```diff
@@ -1,5 +1,6 @@
+# Managed by puppet (nebula/profile/hathitrust/solr6/log4j.properties.erb)
 #  Logging level
-solr.log=${solr.solr.home}/../logs
+solr.log=<%= @solr_home %>/logs
```

Comment:
Might as well -- reduces the indirection and makes this much clearer when actually looking at the realized config